website/docs/d/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_provider_snapshot_backup_policy"
+sidebar_current: "docs-mongodbatlas-datasource-cloud-provider-snapshot-backup-policy"
+description: |-
+    Provides an Cloud Provider Snapshot Backup Policy Datasource.
```

Comment:
Change An to A:
     Provides **a** Cloud Provider Snapshot Backup Policy Datasource.